easybuild/easyconfigs/o/OpenCV/OpenCV-2.4.9-intel-2015a.eb

```diff
@@ -0,0 +1,80 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenCV'
+version = '2.4.9'
+
+homepage = 'http://opencv.org/'
+description = """OpenCV (Open Source Computer Vision Library) is an open source computer vision
+ and machine learning software library. OpenCV was built to provide
+ a common infrastructure for computer vision applications and to accelerate
+ the use of machine perception in the commercial products."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'opt': True, 'optarch': True}
+
+sources = [SOURCELOWER_ZIP]
+source_urls = [('http://sourceforge.net/projects/opencvlibrary/files/opencv-unix/%(version)s', 'download')]
+
+patches = ['OpenCV-%(version)s_with_IPP.patch']
+
+osdependencies = ['gtk2-devel']
+
+builddependencies = [
+    ('CMake', '3.0.0'),
+]
+
+python = 'Python'
+pythonver = '2.7.9'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+java = 'Java'
+javaver = '1.7.0_51'
+
+dependencies = [
+    (python, pythonver),
+    ('CMake', '3.0.0'),
+    ('zlib', '1.2.8'),
+    ('ffmpeg', '2.4'),
+    ('libjpeg-turbo', '1.3.1'),
+    ('libpng', '1.6.12'),
+    ('LibTIFF', '4.0.3'),
+    ('JasPer', '1.900.1'),
+    (java, javaver ,'', True),
+    ('ant', '1.9.3', '-%s-%s' % (java,javaver), True),
+    ('GLib', '2.40.0')
+]
+
+preconfigopts = 'export IPPROOT=$EBROOTICC/ipp && '
+
+configopts = '-DCMAKE_BUILD_TYPE=RELEASE '
+configopts += '-DBUILD_PYTHON_SUPPORT=ON '
+configopts += '-DPYTHON_PACKAGES_PATH=$EBROOTPYTHON/lib/python%s/site-packages ' % pythonshortver
+configopts += '-DPYTHON_EXECUTABLE=$EBROOTPYTHON/bin/python '
+configopts += '-DPYTHON_LIBRARIES=$EBROOTPYTHON/lib '
+configopts += '-DPYTHON_INCLUDE_PATH=$EBROOTPYTHON/include/python%s ' % pythonshortver
```

Comment:
these `-DPYTHON_*` are new, they should be added to the other OpenCV easyconfigs too?